subsys/bluetooth/controller/ll_sw/ull_central_iso.c

```diff
@@ -405,28 +416,47 @@ uint8_t ll_cig_parameters_commit(uint8_t cig_id, uint16_t *handles)
 			cis->lll.rx.ft = cis->lll.tx.ft;
 
 #elif defined(CONFIG_BT_CTLR_CONN_ISO_RELIABILITY_POLICY)
-			/* Utilize Max_Transmission_latency */
-			if (cis->framed) {
-				/* TL = CIG_Sync_Delay + FT x ISO_Interval + SDU_Interval.
-				 * SDU_Interval <= CIG_Sync_Delay
-				 */
-				cis->lll.tx.ft =
-					DIV_ROUND_UP(cig->c_latency - cig->c_sdu_interval -
-							iso_interval_us, iso_interval_us);
-				cis->lll.rx.ft =
-					DIV_ROUND_UP(cig->p_latency - cig->p_sdu_interval -
-							iso_interval_us, iso_interval_us);
-			} else {
-				/* TL = CIG_Sync_Delay + FT x ISO_Interval - SDU_Interval.
-				 * SDU_Interval <= CIG_Sync_Delay
-				 */
-				cis->lll.tx.ft =
-					DIV_ROUND_UP(cig->c_latency + cig->c_sdu_interval -
-							iso_interval_us, iso_interval_us);
-				cis->lll.rx.ft =
-					DIV_ROUND_UP(cig->p_latency + cig->p_sdu_interval -
-							iso_interval_us, iso_interval_us);
+			/* Utilize Max_Transport_latency */
+
+			/* Framed:
+			 *   TL = CIG_Sync_Delay + FT x ISO_Interval + SDU_Interval
+			 *
+			 * Unframed:
+			 *   TL = CIG_Sync_Delay + FT x ISO_Interval - SDU_Interval
+			 *
+			 * Set CIG_Sync_Delay = ISO_Interval as largest possible CIG_Sync_Delay.
+			 * This favors utilizing as much as possible of the Max_Transport_latency,
+			 * and spreads out payloads over multiple CIS events (if necessary).
+			 */
+			uint32_t cig_sync_delay_max = iso_interval_us;
```

Comment:
```suggestion
			uint32_t cig_sync_delay_us_max = iso_interval_us;
```

Suggest to keep the unit in the variable name